src/database/queries/questions.js

```diff
@@ -0,0 +1,80 @@
+const Question = require('../models/index').Question
+
+module.exports = class QuestionsData {
+	static async createQuestion(data) {
+		try {
+			const question = await Question.create(data, { returning: true })
+			return question
+		} catch (error) {
+			return error
+		}
+	}
+
+	static async findOneQuestion(filter, projection = {}) {
+		try {
+			const questionData = await Question.findOne({
+				where: filter,
+				attributes: projection,
+				raw: true,
+			})
+			return questionData
+		} catch (error) {
+			return error
+		}
+	}
+
+	static async find(filter, projection = {}) {
+		try {
+			const questionData = await Question.findAll({
+				where: filter,
+				attributes: projection,
+				raw: true,
+			})
+			return questionData
+		} catch (error) {
+			return error
+		}
+	}
+
+	static async updateOneQuestion(filter, update, options = {}) {
+		try {
+			const [rowsAffected] = await Question.update(update, {
+				where: filter,
+				...options,
+			})
+
+			return rowsAffected > 0 ? 'QUESTION_UPDATED' : 'QUESTION_NOT_FOUND'
```

Comment:
Note to Priyanka... If not the affectedRowsCount, we can use this pattern. But let's keep on pattern across services.